stage1/init/common/pod.go

```diff
@@ -1046,3 +1052,26 @@ func parseLinuxCapabilitiesSet(capSet types.LinuxCapabilitiesSet) []string {
 	}
 	return capsStr
 }
+
+func getAppNoNewPrivileges(isolators types.Isolators) (bool, error) {
+	noNewPrivs := ""
+
+	// TODO(sur): once https://github.com/appc/spec/pull/611 lands, bump appc spec, and use official appc types instead
+	for _, isolator := range isolators {
+		if isolator.Name == "os/linux/no-new-privs" {
+			noNewPrivs = string(*isolator.ValueRaw)
+			break
```

Comment:
If there are several isolators of type "os/linux/no-new-privs", we should take the most restrictive one.

See also getAppCapabilities(), where we take the union of isolator caps.

Shouldn't we also return an error if the value is neither true or false?